Public/Src/Cache/ContentStore/Grpc/ContentStore.proto

```diff
@@ -2,8 +2,8 @@
 
 package ContentStore.Grpc;
 
-// Defines an interface of a content server
-service ContentServer {
+// Defines an interface of a cache server
+service CacheServer {

```

Comment:
I think this is breaking change on the protocol level and it means that when the new grpc client will try to use an old casaas they will fail.

Reply:
Yes, that is the case indeed. I actually just checked that the current implementation only fails when an old client connects to a new server (locally)

Reply:
Fixed by still providing the old implementation as well as the current one. It's a temporary transition plan that should be in place until all stamps have the bits shipped